### PR TITLE
Create community-playbook.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added community playbook to Handbook.

### CODE REVIEW

File: contributors/contributors/community-playbook.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
+# Community Playbook
+
+This guide outlines the process of running community campaigns.
```
Comment (jasonblais) [inline contributors/contributors/community-playbook.md]:
```suggestion
This playbook outlines various processes and practices used in the Mattermost community, including running community design meetings and campaigns, managing GitHub and forum issues, and managing Twitter.
```

**Review by jasonblais — DISMISSED** [review]
One suggestion to the opening line, else LGTM!

**Comment by jasonblais** [comment]
As an aside, the page is becoming long and would likely be easier to navigate through shorter subpages. But that can be done separately when time.

**Comment by justinegeffen** [comment]
> As an aside, the page is becoming long and would likely be easier to navigate through shorter subpages. But that can be done separately when time.

I'm 5/5 on that. One difference between the docs version and this one is that this one has a RHS nav and the headers are easier to locate. So that's a happy medium for now. Ideally this would be broken up into individual sections and nested under the landing page. I'll address that down the line if that works for you.

**Review by jasonblais — APPROVED** [review]

**Pull request merged**